app/build.gradle

```diff
@@ -35,7 +35,10 @@ android {
 
   productFlavors {
 
-    playground {} // Reserved for development
+    playground {
+      // Reserved for development
+      ext.enableCrashlytics = false
+    }
```

Comment:
I thought you only wanted it for droidconBerlin and hence I deactivated it for playground.